### ISSUE TITLE
channel is in TRYOPEN instead of OPEN

### ISSUE THREAD

**Issue by greg-szabo** [informalsystems/ibc-rs#1074]
## Crate
relayer
relayer-cli

## Summary of Bug
Using hermes built from master during the start command the following is received in the log for all chains:

```
Jun 10 22:10:59.225 DEBUG ibc_relayer::supervisor: channel channel-0 on chain ibc-2 is: TRYOPEN; state on dest. chain (ibc-0) is: OPEN
Jun 10 22:10:59.225 DEBUG ibc_relayer::supervisor: done spawning workers for chain ibc-2 and channel channel-0
Jun 10 22:10:59.233 DEBUG ibc_relayer::supervisor: channel channel-0 on chain ibc-3 is: TRYOPEN; state on dest. chain (ibc-0) is: OPEN
Jun 10 22:10:59.233 DEBUG ibc_relayer::supervisor: done spawning workers for chain ibc-3 and channel channel-0
Jun 10 22:10:59.243 DEBUG ibc_relayer::supervisor: channel channel-0 on chain ibc-4 is: TRYOPEN; state on dest. chain (ibc-0) is: OPEN
Jun 10 22:10:59.243 DEBUG ibc_relayer::supervisor: done spawning workers for chain ibc-4 and channel channel-0
Jun 10 22:10:59.250 DEBUG ibc_relayer::supervisor: channel channel-0 on chain ibc-5 is: TRYOPEN; state on dest. chain (ibc-0) is: OPEN
Jun 10 22:10:59.250 DEBUG ibc_relayer::supervisor: done spawning workers for chain ibc-5 and channel channel-0
Jun 10 22:10:59.257 DEBUG ibc_relayer::supervisor: channel channel-0 on chain ibc-6 is: TRYOPEN; state on dest. chain (ibc-0) is: OPEN
Jun 10 22:10:59.257 DEBUG ibc_relayer::supervisor: done spawning workers for chain ibc-6 and channel channel-0
```

## Version
master branch, `3b673a3065c8395fd3a8933255a4531e93ddfea4` Add --hd-path option to keys restore and keys add commands (#1061)

## Steps to Reproduce
gm.toml: (available on the testnet server)
```
[global]
  add_to_hermes = true
  gaiad_binary = "/opt/bin/gaiad_4.2.1"
  ports_start_at = 27070

  [global.hermes]
    binary = "/opt/bin/hermes_0.4.0_post"
    log_level = "trace"

[ibc-0]
  ports_start_at = 27000

[ibc-1]
  gaiad_binary = "/opt/bin/regen_1.1.0-alpha1"
  ports_start_at = 27010

[ibc-2]
  gaiad_binary = "/opt/bin/persistenceCore_0.1.3"
  hdpath = "m/44'/750'/0'/0/0"
  ports_start_at = 27020

[ibc-3]
  gaiad_binary = "/opt/bin/iris_1.1.1"
  ports_start_at = 27030

[ibc-4]
  gaiad_binary = "/opt/bin/akash_0.9.4-rc9"
  ports_start_at = 27040

[ibc-5]
  gaiad_binary = "/opt/bin/regen_1.0.0"
  ports_start_at = 27050

[ibc-6]
  gaiad_binary = "/opt/bin/chain-maind"
  hdpath = "m/44'/394'/0'/0/1"
  ports_start_at = 27060
```

```
gm start
gm hermes config
gm hermes keys
gm hermes cc
# Take all createclients from ibc0 to everywhere else and execute them. They will succeed.
hermes start
```

You can `cat nohup.out` on the testnet server to see the hermes output. No transactions were submitted on the chains apart from the `hermes create-client` command.

## Acceptance Criteria
Done: No error message, as it was with the `hermes 0.4.0` release.

____

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate milestone (priority) applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by adizere** [comment]
Thanks Greg! And apologies, we should have told you about the potential problem with master. I think the issue you're seeing is with a bug in the `channel create` command, see #1064. That issue is being fixed in #1066.

If you can confirm that you were indeed relying on `create channel` command, then I think we can file this as a duplicate of #1064.

**Comment by adizere** [comment]
dup of #1064
